sdk/instruction/src/lib.rs

```diff
@@ -26,6 +26,8 @@ pub use account_meta::AccountMeta;
 pub mod error;
 #[cfg(target_os = "solana")]
 pub mod syscalls;
+#[cfg(feature = "std")]
```

Comment:
nit: there's an inconsistency between having this `#[cfg` over the mod and then having another `#![cfg` in the file. I think we've typically put them over the mod declaration, so let's remove `#![cfg(target_arch = "wasm32")]` from wasm.rs and change this to
```suggestion
#[cfg(all(feature = "std", target_arch = "wasm32"))]
```